src/java/org/apache/cassandra/db/compaction/LeveledManifest.java

```diff
@@ -164,8 +164,7 @@ public synchronized void add(SSTableReader reader)
             try
             {
                 logger.debug("Could not add sstable {} in level {} - dropping to 0", reader, reader.getSSTableLevel());
-                reader.descriptor.getMetadataSerializer().mutateLevel(reader.descriptor, 0);
-                reader.reloadSSTableMetadata();
+                reader.mutateAndReloadStats(stats -> stats.mutateLevel(0));
```

Comment:
Same as above. Does this mean we no longer trace like we would have in `IMetadataSerializer#mutateLevel()`?

Reply:
reverted to preload stats component